prow/plugins/label/label.go

```diff
@@ -226,19 +230,26 @@ func handle(gc githubClient, log *logrus.Entry, ae assignEvent) error {
 	}
 
 	toRepeat := []string{}
+	isMember := false
 	if len(sigMatches) > 0 {
-		isMember, _ := gc.IsMember(ae.org, ae.login)
-		if !isMember {
-			toRepeat = ae.getRepeats(sigMatches, existingLabels)
-		}
+		isMember, _ = gc.IsMember(ae.org, ae.login)
+		toRepeat = ae.getRepeats(sigMatches, existingLabels)
 	}
 
-	if len(toRepeat) > 0 {
+	if (len(toRepeat) > 0) || !isMember {
```

Comment:
I think this is still a little off. The way it is now, a comment will be created to reiterate the mention even if the original mention was made by an org member.  Additionally, a comment will be created whenever a non-member leaves a comment, even if they didn't leave any mentions.  I think something like the following would work though:
```
if len(toRepeat) > 0 {
                if !isMember {
  		        msg := fmt.Sprintf(chatBack, strings.Join(toRepeat, ", "))
  		        if err := gc.CreateComment(ae.org, ae.repo, ae.number, plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg)); err != nil {
  			        log.WithError(err).Errorf("Could not create comment \"%s\".", msg)
  		        }
                }
  
 		if sc != nil {
 			//if sig matches then send a notification on slack
 			for _, sig := range sigMatches {
 				if err := sc.WriteMessage(plugins.FormatResponseRaw(ae.body, ae.url, ae.login, msg), "sig-" sig[1]); err != nil {
 					log.WithError(err).Error("Failed to send message on slack channel: ", "sig-"+sig[1], " with message ", msg)
 				}
 			}
  		}
  	}
```